src/styles/main.scss

```
@@ -1,3 +1,23 @@
-body {
+@import "./header.scss";
+@import "./cards.scss";
+@import "./stars.scss";
+@import "./variables.scss";
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+}
```

```suggestion
* {
  box-sizing: border-box;
}
```
it's forbidden to use anything except `box-sizing `in `*`